### PR TITLE
initial model test with validations to user

### PR DESCRIPTION
**Pull request by Weltinio**
- Added first_name, last_name, and date-of-birth columns to Users
 image
- Made validations for the added columns
- created rspec tests for the added columns' validations
- need to figure out the issue with age validation.
image

### CODE REVIEW

File: spec/factories/user.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    email {"test@email.com"}
+    password {"123456"}
+		first_name { "Test" }
+		last_name { "Tester" }
+		age { 19 }
+  end
+end
```
Comment (analyn34) [inline spec/factories/user.rb:9]: fix code alignment

**Comment by analyn34** [comment]
hello @Weltinio , kindly fix your test failure first, you may add a `WIP` label in your PR to avoid merging an unfinished PR

File: spec/factories/user.rb (hunk 1 above)
Comment (Weltinio) [reply spec/factories/user.rb:9]: image
This is what mine looks like.

**Comment by Weltinio** [comment]
> hello @Weltinio , kindly fix your failure test first, you may add a `WIP` label in your PR to avoid merging an unfinished PR

Got it!

**Pull request closed without merging**